Plugins/Events/NWScript/nwnx_events.nss

```diff
@@ -893,6 +893,19 @@ _______________________________________
     THIEVES_TOOL          | object | Convert to object with NWNX_Object_StringToObject()
     ACTIVE_PROPERTY_INDEX | int    |
 
+_______________________________________
+    ## UUID Collision Events
+    - NWNX_ON_UUID_COLLISION_BEFORE
+    - NWNX_ON_UUID_COLLISION_AFTER
+
+    `OBJECT_SELF` = The object that caused the UUID collision
+
+    Event Data Tag        | Type   | Notes
+    ----------------------|--------|-------
+    UUID                  | string | The UUID
+
+    Note: To get the existing object with `UUID` you can use GetObjectByUUID()
```

Comment:
IIUC, this will fire before the object is actually added to the world, so many functions won't work (like `GetArea(OBJECT_SELF)`). If that's the case, would be nice to note.